### DISCUSSION TITLE
Environment variables type with Next.js

### DISCUSSION THREAD

**Discussion by kopyl** [vercel/community#209] [Help] [answered]
Which environment variables uses React? Ones accessed from `process.env`, correct?

But which environment variables uses Next.js? Ones accessed from `printenv` command from OS terminal?

**Comment by samcx** [comment, marked as answer]
Environment Variables don't use React or Next.js to be exact.

For more information on using Environment Variables with Next.js → https://nextjs.org/docs/basic-features/environment-variables.

**Reply by kopyl** [reply]
But they can be replaced with some on a build, correct?

**Reply by samcx** [reply]
Can you clarify what you mean by this?
